R/metab.bookkeep.R

```diff
@@ -113,7 +113,7 @@ metab.bookkeep <- function(do.obs, do.sat, k.gas, z.mix, irr, ...){
 
 	R <- mean(nep.night, na.rm=TRUE) * freq # should be negative
 	NEP <- mean(delta.do.metab, na.rm=TRUE) * freq # can be positive or negative
-	GPP <- mean(nep.day, na.rm=TRUE) * sum(dayI) - R # should be positive
+	GPP <- mean(nep.day, na.rm = TRUE) * sum(dayI) - mean(nep.night, na.rm = TRUE) * sum(dayI) # should be positive
```

Comment:
I like the `GPP <- NEP - R` formulation. It might also not produce the same answer due to differences in `freq` and `dayI` + `nightI`. Maybe another reason to use this formulation to be consistent with the usage of `freq` in NEP and R estimations. 

`sum(nep.day)` should give the same answer as `mean(nep.day) * sum(dayI)`